e2e/transfer_test.go

```diff
@@ -0,0 +1,77 @@
+package e2e
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/strangelove-ventures/ibctest/ibc"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/cosmos/ibc-go/e2e/testsuite"
+	"github.com/cosmos/ibc-go/e2e/testvalues"
+)
+
+func TestTransferTestSuite(t *testing.T) {
+	suite.Run(t, new(TransferTestSuite))
+}
+
+type TransferTestSuite struct {
+	testsuite.E2ETestSuite
+}
+
+func (s *TransferTestSuite) TestMsgTransfer_Timeout_Nonincentivized() {
+	t := s.T()
+	ctx := context.TODO()
+
+	relayer, channelA := s.SetupChainsRelayerAndChannel(ctx, transferChannelOptions())
+	chainA, chainB := s.GetChains()
+	chainAWallet := s.CreateUserOnChainA(ctx, testvalues.StartingTokenAmount)
+	chainBWallet := s.CreateUserOnChainB(ctx, testvalues.StartingTokenAmount)
+
+	chainBWalletAmount := ibc.WalletAmount{
+		Address: chainBWallet.Bech32Address(chainB.Config().Bech32Prefix), // destination address
+		Denom:   chainA.Config().Denom,
+		Amount:  testvalues.IBCTransferAmount,
+	}
+
+	t.Run("IBC transfer packet timesout", func(t *testing.T) {
+		tx, err := chainA.SendIBCTransfer(ctx, channelA.ChannelID, chainAWallet.KeyName, chainBWalletAmount, testvalues.ImmediatelyTimeout())
```

Comment:
@colin-axner I tried to use your [helper function](https://github.com/cosmos/ibc-go/blob/acda77b44fd9be710bb498a5c033eaba27df8f12/e2e/transfer_test.go#L31) for broadcasting the message instead of using the built-in library fn but for some reason the packet never timesout. 

Not sure if this is a golang relayer issue or if the message doesn't get broadcast correctly.

Reply:
should we create an issue to investigate this?

Reply:
Agreed, I'd like to understand why this is the case?